### PR TITLE
proper description of appcodechecker in config sample

### LINKED ISSUE #14114: Missleading comment in config sample file in OC-8
In the cinfog sample file bundled with OC-8 there is this comment: 

```php
/**
 * Check 3rd party apps to make sure they are using the private API and not the
 * public API. If the app uses the private API it cannot be installed.
 */
```

This sounds funny to me, somehow. No wonder there are so few 3rdparty apps :-)

### PR DESCRIPTION
**Pull request by MorrisJobke**
fixes #14114

cc @carlaschroder @arkascha @PVince81

### CODE REVIEW

**Comment by MorrisJobke** [comment]
@karlitschek backport request to update the stable8 documentation too.

File: config/config.sample.php [hunk 1]
```diff
@@ -420,8 +420,9 @@
  */
 
 /**
- * Check 3rd party apps to make sure they are using the private API and not the
- * public API. If the app uses the private API it cannot be installed.
+ * Checks an app before install if it uses private APIs instead of the proper
```
Comment (PVince81) [inline config/config.sample.php]: if=>whether ? (in german "if" == "wenn" and "whether" == "ob")

**Comment by karlitschek** [comment]
nice. please backport

File: config/config.sample.php [hunk 2]
```diff
@@ -420,8 +420,9 @@
  */
 
 /**
- * Check 3rd party apps to make sure they are using the private API and not the
- * public API. If the app uses the private API it cannot be installed.
+ * Checks an app before install whether it uses private APIs instead of the 
+ * proper public APIs. If this is set to true it will just allow to install or
```
Comment (Xenopathic) [inline config/config.sample.php]: just => only

**Comment by carlaschroder** [comment]
Works for me. Make it so. :+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Comment by MorrisJobke** [comment]
Backport: #15006

File: config/config.sample.php (hunk 2 above)
Comment (MorrisJobke) [inline config/config.sample.php]: bd57fcb

**Pull request merged**